sdjournal/read.go

```diff
@@ -33,6 +33,7 @@ type JournalReaderConfig struct {
 	// where the reading begins within the journal.
```

Comment:
This comment should be updated to reflect that Cursor is also mutually exclusive (would also be good to list that they are tried in order)